package.json

```diff
@@ -93,11 +93,11 @@
     "nodemon": "^3.1.3",
     "patch-package": "^8.0.0",
     "prettier": "2.8.8",
-    "react": "18.3.1",
-    "react-native": "0.77.1",
+    "react": "^19.0.0",
+    "react-native": "0.78.0",
```

Comment:
React Native 0.78.2 is out, let's bump to 0.78.2 if possible: https://react-native-community.github.io/upgrade-helper/?from=0.78.0&to=0.78.2

Reply:
Done: https://github.com/Expensify/react-native-live-markdown/pull/649/commits/e424f664589b68000f7bc98da08d7757d584638f